.github/workflows/ci.yml

```diff
@@ -763,13 +763,13 @@ jobs:
       - shell: bash
         working-directory: s2nTests
         run: |
-          docker-compose pull
+          docker compose pull --ignore-buildable
```

Comment:
Can you please document the need for this flag in our GitHub_CI best practices?

Reply:
Done.